nuit/utils.py

```diff
@@ -13,10 +13,24 @@ def _closure(fn):
         return fn.__closure__
 
 
+def _code(fn):
```

Comment:
Yes, I actually covered this case later, but forgot to push latest changes. Were investigating some weird issue related to '__func__'